nnvm/Makefile

```diff
@@ -18,7 +18,7 @@ CFLAGS += -I$(TVMPATH)/include -I$(TVMPATH)/3rdparty/dlpack/include -I$(TVMPATH)
 ifdef DMLC_CORE_PATH
   CFLAGS += -I$(DMLC_CORE_PATH)/include
 else
-  CFLAGS += -I$(ROOTDIR)/../dmlc-core/include
+  CFLAGS += -I$(ROOTDIR)/../3rdparty/dmlc-core/include
```

Comment:
-I$(TVMPATH)/3rdparty/dmlc-core/include is better?

Reply:
@siju-samuel Updated. Thanks.